src/redux/items/state.js

```diff
@@ -7,8 +7,8 @@ export const baseItems = [
   {
     key: '1',
     id: 1,
-    image: cricut,
-    name: 'Cricut crafting tool',
+    image: circuit,
```

Comment:
nit: i also thought this was a typo, turns out it's actually Cricut: https://cricut.com/en_us

Reply:
Oh dang hahaha. Confusing

Reply:
oh lol yeah it is cricut. i want this tool soo bad =D